### PR TITLE
Created usecases for addon, food and customer.

### PR DESCRIPTION
**Pull request by VijayS02**

### CODE REVIEW

**Review by akashilangovan — APPROVED** [review]
Looks good to me

File: src/businessrules/food/usecases/UpdateFoodUseCase.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        Food food = FoodLoader.loadFoodFromId(FoodId);
+
+        if(vendor == null || food == null){
+            errorHandler.displayError("Incorrect vendorToken or FoodId.");
+            return false;
+        }
+
+
+        Shop shop = vendor.getShop();
+        boolean success = shop.getMenu().updateFood(FoodId, food);
```
[the remark this answers is not in the document]
Comment (VijayS02) [reply src/businessrules/food/usecases/UpdateFoodUseCase.java:49]: Youre right, my bad on that one.

[the remark this answers is not in the document]
Comment (VijayS02) [reply src/businessrules/food/usecases/UpdateFoodUseCase.java]: Please check my fix.

**Review by akashilangovan — APPROVED** [review]
lgtm

File: src/businessrules/food/usecases/UpdateFoodUseCase.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (stephlum) [reply src/businessrules/food/usecases/UpdateFoodUseCase.java]: When you call updateFood shouldnt that be with the new object (i.e. like
`boolean success = shop.getMenu().updateFood(FoodId, FoodLoader.loadFood(object))`

**Review by stephlum — COMMENTED** [review]
please see comment about same issue as previously

**Review by jeremiahwong007 — APPROVED** [review]
Looks good.

**Pull request merged**